src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -196,7 +218,21 @@ NavigationAccordion.propTypes = {
       id: PropTypes.string.isRequired,
       tag: PropTypes.string,
       hideSubpages: PropTypes.bool, // needed for /help/tutorials and /help/troublehshooting
-      pages: PropTypes.array
+      pages: PropTypes.arrayOf(
+        PropTypes.shape({
+          title: PropTypes.string.isRequired,
+          path: PropTypes.string.isRequired,
+          id: PropTypes.string.isRequired,
```

Comment:
I'm not sure the `id` is required for all pages, maybe just subpages? Can you remind me why subpages have an `id`?

<img width="366" alt="2021-06-30 at 10 46 AM" src="https://user-images.githubusercontent.com/2180540/123981706-77998580-d990-11eb-8b99-5559fbb4efc9.png">

Reply:
Oh I think I misunderstood your comment above. https://github.com/mapbox/dr-ui/pull/453#discussion_r647511686 😕